target-chains/near/receiver/workspace-test.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# This script is used to prepare the environment in order to run the NEAR
+# workspaces based tests. It relies on the relative position of the wormhole-
+# stub contract to this directory.
+set -euo pipefail
+
+# Setup rust to build wasm.
+rustup target add wasm32-unknown-unknown
+
+cargo build --release --target wasm32-unknown-unknown
+cp target/wasm32-unknown-unknown/release/pyth.wasm .
+
+(
+    cd ../wormhole-stub
+    cargo build --release --target wasm32-unknown-unknown
+    cp target/wasm32-unknown-unknown/release/wormhole_stub.wasm ../receiver
+)
+
+RUST_LOG=info cargo nextest run
```

Comment:
Can you document that nextest is required?

Reply:
Also it would be awesome if you add this to CI

Reply:
Also I see the wasm files are not gitignored. it would be great to have them in gitignore too.